next.config.example.js

```diff
@@ -18,23 +18,42 @@ module.exports = {
                 // api: 'https://your.etherpadserver.org/mypads/api',
                 // spacesGroupName: '/spaces'
             },
+            spacedeck: {
+                path: '/sketch',
+                baseUrl: 'http://spacedeck.localhost',
```

Comment:
Can we comment out spacedeck by default?
Similar to how I commented out the `etherpad.api` property -- because it's not a real MyPads API -- I think I'd like to comment this out to make sure that `next.config.example.js` is a _working_ config, if someone would just copy and paste it to `next.config.js` exactly as it is.

Following that logic I guess that `matrixContentStorage` would have to be commented out as well...

What do you think @marcel-klasse?